Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Build the manager binary
-FROM golang:1.18 as builder
+FROM golang:1.20 as builder
```

Comment:
I am ok with this change but let's prefer to roll out version upgrades as separate changes for better trackability.